src/test/java/org/folio/rest/impl/FeeFineActionsAPITest.java

```diff
@@ -338,12 +338,18 @@ public void postActionWithPatronNotice() {
           .put("copy", item.getCopyNumber())
           .put("numberOfPieces", item.getNumberOfPieces())
           .put("descriptionOfPieces", item.getDescriptionOfPieces())
+          .put("accessionNumber", item.getAccessionNumber())
           .put("callNumber", item.getEffectiveCallNumberComponents().getCallNumber())
           .put("callNumberPrefix", item.getEffectiveCallNumberComponents().getPrefix())
           .put("callNumberSuffix", item.getEffectiveCallNumberComponents().getSuffix())
           .put("title", instance.getTitle())
+          .put("instanceHrid", instance.getHrid())
           .put("primaryContributor", PRIMARY_CONTRIBUTOR_NAME)
           .put("allContributors", PRIMARY_CONTRIBUTOR_NAME + "; " + NON_PRIMARY_CONTRIBUTOR_NAME)
+          .put("physicalDescriptions", "Hardback; Green")
+          .put("editions", "1st; 2nd")
```

Comment:
As far as I remember, array of editions is deserialized into a set, which means the order of its elements may change, so you can end up with "1st; 2nd" or "2nd; 1st".

Reply:
All right, testing with list of one.